optimum/exporters/neuron/base.py

```diff
@@ -60,6 +64,8 @@ class NeuronConfig(ExportConfig, ABC):
             The model configuration.
         task (`str`, defaults to `"feature-extraction"`):
             The task the model should be exported for.
+        dynamic (`bool`, defaults to `False`):
```

Comment:
```suggestion
        dynamic_batch_size (`bool`, defaults to `False`):
```

Reply:
ok, I will use `dynamic_batch_size` instead of `dynamic`.